cumulusci/cli/cli.py

```diff
@@ -137,6 +137,16 @@ def render_recursive(data, indent=None):
             else:
                 click.echo('{}{} {}'.format(indent_str, key_str, value))
 
+def check_org_overwrite(config, org_name):
+    try:
+        org = config.keychain.get_org(org_name)
+        raise click.ClickException(
```

Comment:
can we prompt the user if they want to delete and move forward. i usually do want to...